platformio.ini

```diff
@@ -47,7 +48,9 @@ platform = espressif32
 board = esp32dev
 framework = arduino
 monitor_baud = 115200
-upload_speed = 921600
+; On my V1, upload does not works over default 115200

```

Comment:
ok, so we take this out again.

(Tested it on Heltec and TTGOv2, on both 921600 work. These boards have CP2102/CP2104 USB bridge chips, those a specified up to 2 MBit/s. says data sheet.)